File: leapYear.py
def leapYear(year):
    if( year % 400 == 0 or year % 4 == 0 and year % 100 != 0):
        print(str(year) + ' is a leap year')
    else:
        print(str(year) + ' is NOT a leap year')

# test cases
leapYear(1996)
leapYear(2020)
leapYear(2021)

File: leapYear.js
function leapYear(year) {
    if ( year % 400 == 0 || year % 4 == 0 && year % 100 != 0){
        return `${year} + is a leap year`
    } else {
        return `${year} + is NOT a leap year`
    }
}


//test cases 
console.log(leapYear(1996)); //leap
console.log(leapYear(2020)); //leap
console.log(leapYear(2021)); //not leap